quixstreams/app.py

```diff
@@ -665,7 +665,13 @@ def add_source(self, source: BaseSource, topic: Optional[Topic] = None) -> Topic
             Default: the source default
         """
         if not topic:
-            topic = self._topic_manager.register(source.default_topic())
+            # Prefix the name of the default topic generated by the Source
+            # for visibility across all the topics.
+            default_topic = source.default_topic()
+            default_topic = default_topic.__clone__(
```

Comment:
why clone? is Topic immutable?

Reply:
Technically, it's not immutable, but it seems cleaner overall. It's done similarly in `QuixTopicManager._finalize_topic`.